Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -517,8 +521,8 @@ extension AutofillUserScript {
             } else {
                 guard let autofillWebsiteAccountMatcher = self.vaultDelegate?.autofillWebsiteAccountMatcher else {
                     credentials = accounts.compactMap {
-                        guard let id = $0.id else { return nil }
-                        return CredentialObject(id: id, username: $0.username, credentialsProvider: credentialsProvider.name.rawValue)
+                        guard let id = $0.id, let username = $0.username else { return nil }
```

Comment:
I wonder, since we accept empty username, maybe we should rather default to "" rather than guard here?

Reply:
Good point. At first I wasn't sure about nullifying username in CredentialObjects, it's a struct used by Autofill User Script so perhaps there's some JS code that expects it to be non-nil. But we could definitely use empty string here. I'll update it 👍